server/dashboard.js

```diff
@@ -24,15 +24,29 @@ const newLocationTemplate = fs.readFileSync(`${__dirname}/mustache-templates/new
 const updateClientTemplate = fs.readFileSync(`${__dirname}/mustache-templates/updateClient.mst`, 'utf-8')
 const updateLocationTemplate = fs.readFileSync(`${__dirname}/mustache-templates/updateLocation.mst`, 'utf-8')
 const vitalsTemplate = fs.readFileSync(`${__dirname}/mustache-templates/vitals.mst`, 'utf-8')
+let languageMode = 'en'
 
 function getAlertTypeDisplayName(alertType) {
   let displayName = ''
-  if (alertType === ALERT_TYPE.SENSOR_DURATION) {
-    displayName = 'Duration'
-  } else if (alertType === ALERT_TYPE.SENSOR_STILLNESS) {
-    displayName = 'Stillness'
+
+  if (languageMode === 'en') {
+    if (alertType === ALERT_TYPE.SENSOR_DURATION) {
+      displayName = 'Duration'
+    } else if (alertType === ALERT_TYPE.SENSOR_STILLNESS) {
+      displayName = 'Stillness'
+    } else {
+      displayName = 'Unknown'
+    }
+  } else if (languageMode === 'es_us') {
+    if (alertType === ALERT_TYPE.SENSOR_DURATION) {
+      displayName = 'Duración'
+    } else if (alertType === ALERT_TYPE.SENSOR_STILLNESS) {
+      displayName = 'Tranquilidad'
+    } else {
+      displayName = 'Desconocido'
+    }
```

Comment:
Just a quick note - I don't we need to add the spanish alert types in here, because these values are used internally. This change will impact the Alert Type in the internal sensors dashboard (on the locations details page) and also the message that is logged when we are handling an alert.

Reply:
Interesting point about internal vs external messaging. I believe that we do use this function for external messaging when coming up with the `alertStart` string here: https://github.com/bravetechnologycoop/BraveSensor/blob/16381dd4e0db93ff1b26b7916394d1c79962507f/server/index.js#L73

So by adding a language parameter to this function, you could send the client's lanauge when forming the `alertStart` message but then always use English when using it to for on the Dashboard.